tests/ci/run_build.py

```diff
@@ -28,13 +28,7 @@
 skip_ci_specs = []
 skip_ci_specs_env = os.getenv('SKIP_CI_SPECS')
 if skip_ci_specs_env:
-    if os.getenv('CIRRUS_CI'):
-        # CIRRUS CI splits multi-line evinronment variables
-        # with a '\n'.
-        skip_ci_specs = skip_ci_specs_env.rstrip().split('\n')
-    else:
-        # GitHub Actions with a space.
-        skip_ci_specs = skip_ci_specs_env.rstrip().split(' ')
+    skip_ci_specs = skip_ci_specs_env.rstrip().split('\n')
```

Comment:
I am curious about this change. Has the way CIRRUS_CI splits multi-line environment variables changed.

Reply:
Actually, it might the change of github action, CIRRUS_CI always use the `\n`:

I print the skip_ci_specs_env: 

```
print(skip_ci_specs_env)
print(skip_ci_specs_env.rstrip().split(' '))
print(skip_ci_specs_env.rstrip().split('\n'))
```

https://github.com/openhpc/ohpc/actions/runs/4239256449/jobs/7367128993#step:8:20

```
components/fs/lustre-client/SPECS/lustre.spec
components/parallel-libs/trilinos/SPECS/trilinos.spec

['components/fs/lustre-client/SPECS/lustre.spec\ncomponents/parallel-libs/trilinos/SPECS/trilinos.spec']
['components/fs/lustre-client/SPECS/lustre.spec', 'components/parallel-libs/trilinos/SPECS/trilinos.spec']
```

It's `components/fs/lustre-client/SPECS/lustre.spec\ncomponents/parallel-libs/trilinos/SPECS/trilinos.spec`, so the space style seems outdate or it was wrong.

Reply:
okay, thanks for the clarification